scripts/generate.sh

```diff
@@ -261,6 +261,9 @@ dart () {
     -c ./config/client/dart.yml.proc.yml
 
   cp "LICENSE" "clients/${PROJECT}/dart"
+
+  # Generate the `*.g.dart` files
+  (cd $dir; command dart run build_runner build)
```

Comment:
The function itself is called `dart` as well. `command` uses the dart executable from PATH. Pretty cool.